### ISSUE TITLE
Syntax error in load_ontology_purge_safe

### ISSUE THREAD

**Issue by nathanweeks** [geneontology/amigo#690]
`npx gulp load-ontology-purge-safe` results in a shell syntax error (`/bin/sh: 1: Syntax error: Unterminated quoted string`), due to a stray leftover single-quote at this line:

https://github.com/geneontology/amigo/blob/6e83d501812b34ad1e4bca99be5fbf725fbee893/gulpfile.js#L427

**Comment by kltm** [comment]
@pkalita-lbl I think this was just a typo, if not let me know; fixed with https://github.com/geneontology/amigo/commit/fa96014c1ea3c1f9cb7de13442fc3caeea29bc10

**Comment by kltm** [comment]
@nathanweeks I'm honestly surprised anybody else is in there to notice the issue. I believe it have been fixed in `master`.

**Comment by pkalita-lbl** [comment]
Yeah that quote looks like refactoring cruft. The fix looks right.
